lib/chronik/projection.ex

```diff
@@ -206,11 +207,20 @@ defmodule Chronik.Projection do
   end
 
   # Replay events from the store.
+  defp fetch_and_replay(:current, state, projection, store) do
+    # TODO: Query the store for only the version, not the whole records!
+    case store.fetch() do
+      {:ok, new_version, _records} ->
+          log(projection, "skipping re-playing")
+          {new_version, state}
+    end
+  end
   defp fetch_and_replay(version, state, projection, store) do
-    case store.fetch(version) do
+    from = if version == :empty do :all else version end
```

Comment:
Probably this fits better in an onliner: `if version == :empty, do: :all, else: version`

Reply:
It is still an one-liner but I' ll change it anyway.